android/app/src/main/java/org/electroncash/electroncash3/Dialog.kt

```diff
@@ -254,8 +254,7 @@ abstract class PasswordDialog<Result> : TaskLauncherDialog<Result>() {
     var password: String by notNull()
 
     override fun onBuildDialog(builder: AlertDialog.Builder) {
-        builder.setTitle(R.string.Enter_password)
-            .setView(R.layout.password)
+        builder.setView(R.layout.password)
```

Comment:
This will also affect the other PasswordDialog subclasses: SendPasswordDialog and ShowSeedPasswordDialog. If you only mean to change the layout of the OpenWalletDialog, then it would be better to make a new layout for that dialog, and leave the base class alone.

Reply:
Ouch, good point.  I've reverted my changes to that base class and extended OpenWalletDialog instead, and created a new layout for it as you suggest. Partly to emphasize my point earlier about naming, I've called the new layout "wallet_open.xml" instead of "open_wallet.xml" but I'm completely open to going the other route.

Reply:
Can't OpenWalletDialog still be a subclasss of PasswordDialog, to avoid duplicating all of that code?

Reply:
Uh, of course.  I wanted to set the title on the dialog differently, but given that the base class sets the title, I thought I couldn't use the base class at all.  But now I just re-set the title after letting the base class do its thing. Derp.